connect_ext_ppr/service.py

```diff
@@ -291,3 +298,57 @@ def create_ppr(ppr, user_id, deployment, db, client, logger):
 def validate_configuration(client, deployment, file_data):
     data = get_configuration_from_media(client, deployment.account_id, deployment.id, file_data.id)
     return validate_configuration_schema(data, deployment.product_id)
+
+
+def add_new_deployment_request(db, dr_data, deployment, account_id, logger):
+    try:
+        deployment_request = DeploymentRequest(
+            deployment_id=dr_data.deployment.id,
+            ppr_id=dr_data.ppr.id,
+            manually=dr_data.manually,
+            delegate_l2=dr_data.delegate_l2,
+            created_by=account_id,
+        )
+        db.set_next_verbose(deployment_request, 'deployment_id')
+        db.commit()
```

Comment:
[COD-A] db.commit() will break the transaction. Better to use db.flush() instead